controllers/rollingupgrade_controller.go

```diff
@@ -744,7 +751,7 @@ func (r *RollingUpgradeReconciler) Process(ctx *context.Context,
 func (r *RollingUpgradeReconciler) validateNodesLaunchDefinition(ruObj *upgrademgrv1alpha1.RollingUpgrade) error {
 	//Get ASG launch config
 	var err error
-	err = r.populateAsg(ruObj)
+	err = r.populateAsg(ruObj, false)
```

Comment:
This has to be `true` as we are doing one final check before marking the rolling upgrade CR completed.